devtools/common/src/main/filtered/extensions.json

```diff
@@ -644,5 +644,13 @@
     "groupId": "io.quarkus",
     "artifactId": "quarkus-vertx",
     "guide": "https://quarkus.io/guides/using-vertx"
+  },
+  {
+    "name": "Vault",
+    "labels": [
+      "vault"
```

Comment:
@vsevel, probably worth adding a "security" or some other label which would characterize the vault concept, @gsmet, what do you think ?

Reply:
done